elastic/kibana
# [Fleet] fix updates available when beta integrations are off

## Summary

Fixes https://github.com/elastic/kibana/issues/149470

Loading installed packages with prerelease flag, so that the `Updates available` is not showing prerelease versions when prerelease flag is off.

How to test:
- Install endpoint latest GA (8.6.1)
- Check that there are no updates available on `Installed integrations` tab
- Enable `Show Beta integrations` switch
- Check that there is 1 badge in `Updates available`, showing the latest `Endpoint` `8.7.0-next` version.

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/90178898/214531019-e1971f73-669d-4eda-b968-0d007763ca59.png">


<img width="832" alt="image" src="https://user-images.githubusercontent.com/90178898/214530754-0131e3e0-ae43-47cb-9454-fb4ab72adc2c.png">
